example/App.js

```diff
@@ -76,7 +96,23 @@ class App extends React.Component {
   }
 
   render() {
+    if (this.state.useGoogleMaps) {
+      return this.renderExamples([
+        [StaticMap, 'StaticMap'],
+        [DisplayLatLng, 'Tracking Position (incomplete)'],
+        [ViewsAsMarkers, 'Arbitrary Views as Markers'],
+        [EventListener, 'Events (incomplete)'],
+        [MarkerTypes, 'Image Based Markers'],
+        [DraggableMarkers, 'Draggable Markers'],
+        [Callouts, 'Custom Callouts'],
+        [Overlays, 'Circles, Polygons, and Polylines (ios error)'],
+        [DefaultMarkers, 'Default Markers'],
+        [TakeSnapshot, 'Take Snapshot (incomplete)'],
```

Comment:
Perhaps instead of rendering a different list of examples, what do you think about adding a third element to the array indicating whether or not Google Maps is supported, and I guess also a fourth element indicating google-specific caption?

```js
this.renderExamples([
  [StaticMap, 'Static map', true],
  [DisplayLatLng, 'Tracking Position', true, '(incomplete)'],
  // ...
  [PolylineCreator, 'Polyline Creator', false],
]);
```